src/utility/api.test.js

```diff
@@ -0,0 +1,13 @@
+import { Get } from './api';
+import Url from './url'
+
```

Comment:
Clearly incomplete testing story here, but I'm somewhat less sure how to add generic tests for the mutative HTTP verbs without mocking.

Reply:
we're not mocking in the calendar, mailing list, and roster tests, so perhaps combined we get 'good enough' coverage

Reply:
I suppose I'm satisfied for the time being haha